src/app/cli/CommandLineInterface.java

```diff
@@ -520,6 +541,295 @@ public void launch(){
                     rate(tokens[1], tokens[2]);
                 }//add more cases
 
+                case PLAY -> {
+                    if(tokens.length <3){
+                        System.out.println("\nUsage:");
+                        System.out.println("PLAY { game | time }");
+                        System.out.println("game            game to be played");
+                        System.out.println("time            time in milliseconds to be added\n"); //Might change later to a more useful format that milliseconds
+                        continue;
+                    }
+                    Game[] game_list = app.search_game_name(tokens[1]);
+
+                    if(game_list.length == 0) {
+                        System.out.println("There is not a game with that name.");
+                        continue;
+                    }
+
+                    Game selected_game = game_list[0];
+                    if(game_list.length > 1) {
+                        System.out.println("Which game would you like to play (enter the number)?");
+                        for(int i = 0; i < game_list.length; ++i) {
+                            Game curr_game = game_list[i];
+                            System.out.println(i+1 + ".\tName: " + curr_game.title() + "\n\tPlay time: " +
+                                    curr_game.playtime());
+                        }
+                        input = in.nextLine();
+                        int input_to_int = Integer.parseInt(input);
+                        selected_game = game_list[input_to_int-1];
+                    }
+                    Time run_time = new Time(Integer.parseInt(tokens[2]));
+                    app.play(selected_game,run_time);
+                }
+
+                case SEARCH_USER -> {
+                    if(tokens.length <2){
+                        System.out.println("\nUsage:");
+                        System.out.println("SEARCH_USER { email }");
+                        System.out.println("email           email that the account is associated with to be searched\n");
+                        continue;
+                    }
+                    User[] user_list = app.search_users(tokens[1]);
+
+                    if(user_list.length==0) {
+                        System.out.println("There no users linked to this email address");
+                        continue;
+                    }
+                    if(user_list.length > 1) {
```

Comment:
the case where `user_list.length == 1` is missed; consider just moving the code inside this `if` block outside of it and removing the `if` itself